keps/349-all-or-nothing/README.md

```diff
@@ -253,11 +271,20 @@ condition, so the corresponding job is unsuspended without further waiting.
 
 ### Timeout on reaching the PodsReady condition
 
-We introduce a timeout, defined in the `waitForPodsReady.timeoutSeconds` field, on reaching the `PodsReady` condition since the job
-is unsuspended (the time of unsuspending a job is marked by the Job's
-`job.status.startTime` field). When the timeout is exceeded, the Kueue's Job
+We introduce two timeouts defined in the `waitForPodsReady.timeoutSeconds` and `waitForPodsReady.replacementTimeoutSeconds` fields.
+
+First one tracks the time between job getting unsuspended (the time of unsuspending a job is marked by the Job's
+`job.status.startTime` field) and reaching the `PodsReady=true` condition.
+
+Second one tracks the time between changing `PodsReady` condition to `false` after the job is running and reaching the
+`PodsReady=true` condition once again.
```

Comment:
Before I review the API, let me clarify the duration of this new timeout if my understanding is correctly.
IIUC, exiting, and new timeouts indicate the below durations, right?

- `timeoutSeconds`:

```mermaid
stateDiagram-v2
    [*] --> Suspended=false: Workload is admitted
    Suspended=false --> PodsReady=false
    PodsReady=false --> PodsReady=false: Waiting for the Pods are ready by timeoutSeconds
    PodsReady=false --> Suspended=true(Requeued): Exceeded the timoutSeconds
```

- `replacementTimeoutSeconds`:

```mermaid
stateDiagram-v2
    [*] --> Suspended=false: Workload is admitted
    Suspended=false --> PodsReady=false(1st): Initializing Containers
    PodsReady=false(1st) --> PodsReady=true: Started Coniainers
    PodsReady=true --> PodsReady=false(2nd): Faild Containers
    PodsReady=false(2nd) --> PodsReady=false(2nd): Waiting for replaced Pods are ready by replacementTimeoutSeconds 
    PodsReady=false(2nd) --> Suspended=true(Requeued): Exeeded replacementTimeoutSeconds

```

Reply:
You understand it just right, thank you for the diagrams

Reply:
I see. In that case, could you clarify the detailed condition when the proposed timeout happens in the KEP?

Reply:
Added flow chart describing it, similar to yours